<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/iconos/imagenes/calculadora.jpg</normaloff>:/iconos/imagenes/calculadora.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
  border-radius: 10px;
  background: #B2BEB5;
  padding: 20px;
  margin: 2px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial Rounded MT Std</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 10px;
    background: #89CFF0;
	margin:2px;
	border: 2px solid black;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_punto">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_divison">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_multi">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_suma">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_resta">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_igual">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_0">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Std Extra Bold</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
